packages/core/contracts/cross-chain-oracle/README.md

```diff
@@ -240,10 +240,10 @@ Transfer deployed contract ownership for each of `Registry`, `Store`, `Identifie
 yarn hardhat transfer-owner --network <l2name> --contract <CONTRACT_ADDRESS> --owner <GOVERNOR_SPOKE_ADDRESS>
 ```
 
-Also transfer `GovernorSpoke` contract ownership to multisig:
+Also transfer `Admin_ChildMessenger` contract ownership to multisig:
```

Comment:
this was wrong, as GovernorSpoke is not ownable.

Reply:
nice catch